internal/cmd/keys.go

```diff
@@ -111,24 +118,35 @@ $ infra keys add connector
 				return err
 			}
 
-			var expMsg strings.Builder
-			expMsg.WriteString("This key will expire in ")
-			expMsg.WriteString(format.ExactDuration(options.TTL))
-			if !resp.Expires.Equal(resp.ExtensionDeadline) {
-				expMsg.WriteString(", and must be used every ")
-				expMsg.WriteString(format.ExactDuration(options.ExtensionDeadline))
-				expMsg.WriteString(" to remain valid")
-			}
-			cli.Output("Issued access key %q for %q", resp.Name, userName)
-			cli.Output(expMsg.String())
-			cli.Output("")
+			if !quiet {
```

Comment:
I'd suggest a guard clause here to keep this more readable (and also reduce the diff):

```go
if quiet {
    cli.Output(resp.AccessKey)
    return nil
}

...
```